authors/apps/articles/models.py

```diff
@@ -88,6 +91,13 @@ def average_rating(self):
         ratings = self.scores.all().aggregate(score=Avg("score"))
         return float('%.2f' % (ratings["score"] if ratings['score'] else 0))
 
+    def calculate_reading_time(self):
```

Comment:
@akram256 , I would advice using a richer library/package than this rudimentary approach. Your approach does not take into account things such as `emojis`, `markdowns` and other formatting characters.

Reply:
Give me an example at least, this is what I could manage to understand

Reply:
Worked on it